src/components/map/MapComponent.vue

```diff
@@ -24,6 +24,10 @@ onMounted(async () => {
   appLoaded.value = true
 })
 
+onUnmounted(() => {
```

Comment:
once a component gets destroyed (unmounted) you have to cleanup resources you created (in this case, the vcsApp). otherwise, you will keep on rendering the vcsApp in its container after a hot reload.

Reply:
Thanks, @bkuster  for the explanation. I have added it as a comment.